lib/commands/create/function.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const promptFunctionName = require('../../prompts/function-name');
+const path = require('../../prompts/path');
+
+const functionWithPayloadSchemaTemplate = require('../../templates/function-with-payload-schema');
+const functionWithoutPayloadSchemaTemplate = require('../../templates/function-without-payload-schema');
+const functionSchemaFs = require('../../fs/function-schema');
+
+const functionSourceTemplate = require('../../templates/function-source');
+const functionSourceFs = require('../../fs/function-source');
+
+const functionTestTemplate = require('../../templates/function-test');
+const functionTestFs = require('../../fs/function-test');
+
+const { ensureOptions } = require('../../utils');
+const { Report } = require('../../report');
+
+
+const getPrompts = () => [
+
+	promptFunctionName,
+	{ ...path, message: 'What\'s the function path?', initial: './src/lambda/{functionName}.js' },
+	{
+		name: 'hasClient',
+		type: 'toggle',
+		message: 'Does the function have a client?',
```

Comment:
Tanto en este mensaje como en el siguiente pondria algo asi como "Must your function be invoked with a ...."